components/config/src/config/languages.rs

```diff
@@ -14,11 +14,11 @@ pub struct LanguageOptions {
     pub title: Option<String>,
     /// Description of the site. Defaults to None
     pub description: Option<String>,
-    /// Whether to generate a feed for that language, defaults to `false`
-    pub generate_feed: bool,
-    /// The filename to use for feeds. Used to find the template, too.
-    /// Defaults to "atom.xml", with "rss.xml" also having a template provided out of the box.
-    pub feed_filename: String,
+    /// Whether to generate feeds for that language, defaults to `false`
+    pub generate_feeds: bool,
+    /// The filenames to use for feeds. Used to find the templates, too.
+    /// Defaults to ["atom.xml"], with "rss.xml" also having a template provided out of the box.
+    pub feed_filenames: Vec<String>,
```

Comment:
`feed_filename = "rss.xml"` should still be accepted too for compat. in another project i've solved this with a `serde(deserialize_with)` macro: https://codeberg.org/selfisekai/electron_tasje/src/commit/3d34593c358a1d68741f3c3d0554447f33d9ec16/src/config.rs#L74-L102